src/appleseed.shaders/as_osl_extensions.h.in

```diff
@@ -139,6 +139,21 @@ closure color as_disney(
     float   clearcoat,
     float   clearcoat_gloss) BUILTIN;
 
+//
+// Plastic BSDF, distribution = beckmann | ggx | gtr | std
+
+closure color as_plastic(
+    string  distribution,
+    normal  N,
+    color   specular_color,
+    float   specular_weight,
+    float   specular_roughness,
+    float   specular_falloff,
```

Comment:
we are calling this param highlight_falloff in other closures. I think it would be good to match the name.